hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/ha/OMFailoverProxyProvider.java

```diff
@@ -464,6 +469,19 @@ public synchronized boolean shouldFailover(Exception ex) {
           return false;
         }
       }
+    } else if (unwrappedException instanceof RpcException) {
+      // Do not failover for following exceptions
+      if (unwrappedException instanceof RpcNoSuchMethodException ||
+          unwrappedException instanceof RpcNoSuchProtocolException ||
+          unwrappedException instanceof RPC.VersionMismatch) {
+        return false;
+      }
+      if (unwrappedException.getMessage().contains(
+          "RPC response exceeds maximum data length") ||
```

Comment:
Similar change we might need in SCM also, for OM-SCM protocol. Can we create this as a common utility method, so that it can be used in SCM also.